server/data/quotes.json

```diff
@@ -312,13 +312,17 @@
     "sentence": "The great enemy of foreign language learning is a sense of shame, an inability or unwillingness to become like a child again and let one's inadequacies show."
   },
   {
-    "author": "Ingrid Bengis",
-    "sentence": "For me, words are a form of action, capable of influencing change."
-  },
-  {
-    "author": "Jacob Ludwig Carl Grimm",
-    "sentence": "Our language is also our history."
-  },
+    "author":"Ingrid Bengis",
+    "sentence":"For me, words are a form of action, capable of influencing change."
+ },
+ {
```

Comment:
Could you please check the indentation here

Reply:
OK, have taken another look, seems some of the brackets went out of alignment? Weird. Formatting again has resulted in 13 changes. Hopefully just spaces.